com.unity.render-pipelines.high-definition/Runtime/Material/SharedRTManager.cs

```diff
@@ -160,6 +165,12 @@ public RenderTargetIdentifier[] GetMotionVectorsPassBuffersRTI(FrameSettings fra
             }
         }
 
+        // Function that will return the UAV for quad overshading and vertex density debug modes
+        public RenderTargetIdentifier GetDebugDisplayBufferRTI()
```

Comment:
I'm not sure this is necessary. RTHandle should convert implicitly to RenderTargetIdentifier iirc.

Reply:
I mean, we can just return the RTHandle and be consistent with most of our other APIs. In general the only place we need the RTI is for MRT arrays.